app/controllers/api/articles_controller.rb

```diff
@@ -1,19 +1,43 @@
+# frozen_string_literal: true
+
 class Api::ArticlesController < ApplicationController
   def index
     collection_articles = Article.all
     if collection_articles.empty?
-      render json: { error_message: 'No articles has been found' }, status: 404
+      render json: { message: 'No articles has been found' }, status: 404
     else
       render json: { articles: collection_articles }, status: 200
    end
   end
 
   def show
-    article = Article.find(params[:id]) rescue nil
+    article = begin
+                Article.find(params[:id])
+              rescue StandardError
+                nil
+              end
```

Comment:
I see no reason for a begin rescue block here. Also, is this show action in scope of this story?

Reply:
fixed this issue